packages/udaru-core/test/integration/authorizeOps.test.js

```diff
@@ -181,6 +181,357 @@ lab.experiment('AuthorizeOps', () => {
+          expect(result).to.exist()
+          expect(result.access).to.be.false()
+
+          udaru.policies.delete({id: testPolicy.id, organizationId}, (err, result) => {
+            expect(err).to.not.exist()
+            done()
+          })
+        })
+      })
+    })
+  })
+
+  lab.test('authorize isUserAuthorized - test with udaru:organizationId variable for condition', (done) => {
+    let Condition = { StringEquals: { 'udaru:organizationId': organizationId } }
+
+    let testPolicy = {
+      id: 'userContextPolicy',
+      name: 'User Context Policy',
+      version: '2012-10-17',
+      organizationId: organizationId,
+      statements: testUtils.StatementWithCondition('Allow', ['read'], ['org:documents/$' + '{udaru:userId}'], Condition)
+    }
+
+    udaru.policies.create(testPolicy, (err, result) => {
+      if (err) return done(err)
+
+      udaru.users.replacePolicies({ id: testUserId, policies: ['userContextPolicy'], organizationId }, (err, result) => {
+        if (err) return done(err)
+
+        authorize.isUserAuthorized({ userId: testUserId, resource: 'org:documents/' + testUserId, action: 'read', organizationId }, (err, result) => {
+          if (err) return done(err)
+
+          expect(err).to.not.exist()
+          expect(result).to.exist()
+          expect(result.access).to.be.true()
+
+          udaru.policies.delete({id: testPolicy.id, organizationId}, (err, result) => {
+            expect(err).to.not.exist()
+            done()
+          })
+        })
+      })
+    })
+  })
+
+  lab.test('authorize isUserAuthorized - test with invalid variable for condition', (done) => {
+    let Condition = { StringEquals: { 'udaru:organizationId': 'invalidOrg' } }
+
+    let testPolicy = {
+      id: 'userContextPolicy',
+      name: 'User Context Policy',
+      version: '2012-10-17',
+      organizationId: organizationId,
+      statements: testUtils.StatementWithCondition('Allow', ['read'], ['org:documents/$' + '{udaru:userId}'], Condition)
+    }
+
+    udaru.policies.create(testPolicy, (err, result) => {
+      if (err) return done(err)
+
+      udaru.users.replacePolicies({ id: testUserId, policies: ['userContextPolicy'], organizationId }, (err, result) => {
+        if (err) return done(err)
+
+        authorize.isUserAuthorized({ userId: testUserId, resource: 'org:documents/' + testUserId, action: 'read', organizationId }, (err, result) => {
+          if (err) return done(err)
+
+          expect(err).to.not.exist()
+          expect(result).to.exist()
+          expect(result.access).to.be.false()
+
+          udaru.policies.delete({id: testPolicy.id, organizationId}, (err, result) => {
+            expect(err).to.not.exist()
+            done()
+          })
+        })
+      })
+    })
+  })
+
+  lab.test('authorize isUserAuthorized - test with udaru:organizationId variable for condition', (done) => {
+    let Condition = { StringEquals: { 'udaru:organizationId': organizationId } }
+
+    let testPolicy = {
```

Comment:
linter should complain about the two let's from above

Reply:
never complained when I ran lint etc.  I changed in latest commit in any case